pyface/ui/qt4/tasks/split_editor_area_pane.py

```diff
@@ -127,6 +134,15 @@ def destroy(self):
         for editor in self.editors[:]:
             self.remove_editor(editor)
 
+        if self.control is not None:
```

Comment:
Same here. The connections actually have nothing to do with `control` so if the list is nonempty, we should disconnect all of them.